.github/workflows/ci.yml

```diff
@@ -19,8 +19,8 @@ on:
       - feature/**
 
 jobs:
-  php-cs-fixer:
-    name: PHP-CS-Fixer
+  coding-standards:
+    name: Coding Standards
```

Comment:
This jobs enforces coding standards.

It currently runs `friendsofphp/php-cs-fixer` in a step, but it could use an additional or a different tool, for example, `squizlabs/php_codesniffer` (see #1566).